app/components/Settings/Settings.react.js

```diff
@@ -461,11 +444,18 @@ class Settings extends Component {
                                                 {`. It has been ${timeElapsed}. Check out the
                                                 FAQ while you wait! 📰`}
                                             </p>
+                                            <p>
+                                                {`Logged in as "${this.props.username}"`}
+                                                <br/>
+                                                <a onClick={this.props.logout}>Log Out</a>
+                                            </p>
                                         </div>
                                     ): (
                                         <div>
                                             <p>
                                                 <a onClick={() => window.location.assign('/login')}>Log into Plotly</a>
+                                                <br/>
+                                                Please, login in order to connect your queries to your Plotly account.
```

Comment:
Oh, didn't saw this line earlier. I thought the message only said "Login to Plotly". .. No change required here imo..